### DISCUSSION TITLE
MediaText in component Definition

### DISCUSSION THREAD

**Discussion by rajevar** [GrapesJS/grapesjs#4934] [Q&A] [answered]
Hello,
The following component definition doesn't display @media query in the generated CSS. Can someone help me out please?

{
   "tagName":"p",
   "type":"text",
   "style":{
      "color":"blue"
   },
   "attributes":{
      "class":"text-large"
   },
   "mediaText":"@media (min-width: 420px) {color: green}",
   "content":" Hello GJS"
}

Thank you,

**Comment by artf** [comment, marked as answer]
Hi @rajevar there is no `mediaText` property by default on components, probably you need [`styles`](https://grapesjs.com/docs/modules/Components.html#components-css) in this case
